orchagent/orchdaemon.cpp

```diff
@@ -97,7 +100,9 @@ bool OrchDaemon::init()
 
     string platform = getenv("platform") ? getenv("platform") : "";
 
-    gCrmOrch = new CrmOrch(m_configDb, CFG_CRM_TABLE_NAME);
+    m_events_handle = events_init_publisher("sonic-events-swss");
```

Comment:
Would it be better to declare this as extern in orch.h or in the .cpp files that uses it?

Reply:
> Please have this as a global variable so that we don't have to pass for each orch and change the signature in all places.

This is the right approach. Thanks!

Reply:
extern in the .cpp file that uses it

Reply:
Done